src/config.rs

```diff
@@ -108,9 +120,16 @@ struct RawStyleConfig {
     pub example_variable: RawStyle,
 }
 
+#[derive(Debug, Default, Serialize, Deserialize, PartialEq, Eq)]
+struct RawDisplayConfig {
+    #[serde(default)]
+    pub pager: bool,
```

Comment:
I would rename this to `use_pager`.

```suggestion
    pub use_pager: bool,
```